docs/baking.md

```diff
@@ -68,6 +76,17 @@ Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` pr
 systemd units for running the corresponding binaries in the background, these units
 are orchestrated by the `tezos-baking-<network>` units.
 
+## Using the wizard
+
+If at this point you want to set up the baking instance using the wizard:
+
+```
+tezos-baking-wizard
```

Comment:
Since this wizard is also able to set up only a node it probably makes sense to rename it to `tezos-setup` or `tezos-setup-wizard`.

Given that before this issue gets merged the wizard is not known to the rest of the world I suggest we do it before that (in a separate PR that should be merged before this one).